packages/cli/commands/init.js

```diff
@@ -91,7 +91,7 @@ exports.handler = async options => {
   });
   const env = options.qa ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD;
 
-  if (configPath) {
+  if ((!c && configPath) || (c && configPath === c)) {
```

Comment:
Since the intention here is to exit the command early in cases where a config file already exists, could we switch this to `fs.existsSync(configPath)`?